src/main/java/de/teddy/advancementhunt/actionbar/ActionbarManager.java

```diff
@@ -34,7 +34,8 @@ public void run() {
 
 
                 for(Player player : Bukkit.getOnlinePlayers()) {
-                    AdvancementHunt.getInstance().getMessageManager().sendMessageReplace(player, MessageType.TIME_LEFT,"%minutes%",PlaceholderAPI.setPlaceholders(player, "%advancement_time_remaining%"));
+                    // I think this is what you mean about placeholder....
+                    AdvancementHunt.getInstance().getMessageManager().sendMessageReplace(player, MessageType.TIME_LEFT,"%minutes%",PlaceholderAPI.setPlaceholders(player, "%ah_time_remaining%"));
```

Comment:
Remove `%minutes%` and allow me to put papi placeholders right into the messages,

Reply:
Fixed in new commit